recipes/cern-root/all/conanfile.py

```diff
@@ -112,6 +116,14 @@ def configure(self):
                     )
                 )
 
+    def _enforce_libcxx_requirements(self):
+        compiler = self.settings.compiler
+        libcxx = compiler.get_safe("libcxx")
+        if libcxx != "libstdc++11":
+            raise ConanInvalidConfiguration(
+                '{} requires "compiler.libcxx=libstdc++11".'.format(self.name)
+            )
```

Comment:
It raises for compilers without libcxx like Visual Studio, very likely not what you want.

Reply:
Thanks for pointing that out.

This commit was an attempt to solve the issue with the build failing when including `conan_basic_setup`.
I have placed a more detailed comment on that issue here and potential solutions to it here:
<https://github.com/conan-io/conan-center-index/pull/3732#discussion_r567484902>

Reply:
Marking this conversation as resolved to keep discussion in one place. 

This is being discussed in the comment thread at: <https://github.com/conan-io/conan-center-index/pull/3732#discussion_r567958665>.